test/jdk/java/awt/Focus/ModalDialogInFocusEventTest.java

```diff
@@ -22,11 +22,11 @@
  */
 
 /*
-  test
+  @test
```

Comment:
For some reason, this test was not enabled when it was open sourced, so enable it.
Testing is green on all platforms.